config/holes.toml

```diff
@@ -3034,3 +3034,51 @@ synopsis = 'Print 𝑒 to the first 1,000 decimal places.'
 preamble = '''
 <p>Print 𝑒 (Euler’s number) to the first 1,000 decimal places.
 '''
+['Rijndael S-boxes']
+ category = 'Number theory'
+ experiment = 857
+ links = [
+     { name = 'Wikipedia', url = 'https://en.wikipedia.org/wiki/Rijndael_S-box'},
+ ]
+ preamble = '''
+ <p>Print the <b>normal and inverse Rijndael S-boxes</b> in a series of hex bytes::
+
+ <blockquote>
+       0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f
```

Comment:
should these not have the 0x prefix too?

Reply:
Good point, will fix.